README.md

```diff
@@ -229,7 +229,7 @@ your Carthage framework copying build phase.
 If you use [CocoaPods][] to manage your dependencies, simply add DrawerKit to your `Podfile`:
 
 ```
-pod 'DrawerKit', '~> 0.0.1'
+pod 'DrawerKit', '~> 0.1.1'
```

Comment:
Ditto.

Reply:
Fixed in https://github.com/Babylonpartners/DrawerKit/pull/21/commits/76e2ce3b2c5cb3ad7de5372b9191c080ffd4b67d.